core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -1,6 +1,7 @@
 package fs2
 
 import scala.collection.immutable.{Queue => SQueue}
+import scala.collection.{Seq => GSeq}
```

Comment:
I've noticed that this problem also extends to `IndexedSeq`. Is it necessary to give it the same treatment?